src/ui/dictionarybar.cc

```diff
@@ -72,7 +72,7 @@ void DictionaryBar::setDictionaries( vector< sptr< Dictionary::Class > > const &
     dictActions.append( action );
   }
 
-  setDictionaryIconSize( 21 );
+  setDictionaryIconSize( 200 );
```

Comment:
The entire method of `DictionaryBar::setDictionaryIconSize` should be just deleted because the height of QToolBar is set automatically. Especially on Linux, it can be any value depends on what theme the user is using. The value `21` was set a decade ago for no reason.

Reply:
i removed it as you said, but it seems that more files have references to that method, for instance of one these files is " scanpopup.cc"